statement.go

```diff
@@ -120,6 +120,8 @@ func (stmt *Statement) QuoteTo(writer clause.Writer, field interface{}) {
 				write(v.Raw, stmt.Schema.PrioritizedPrimaryField.DBName)
 			} else if len(stmt.Schema.DBNames) > 0 {
 				write(v.Raw, stmt.Schema.DBNames[0])
+			} else {
+				stmt.DB.AddError(ErrModelValueRequired) //nolint:typecheck,errcheck
```

Comment:
Why we don't use `ErrPrimaryKeyRequired`.

I am not very familiar with gorm. Maybe I asked a stupid question.

I am not very sure what the meaning `ErrPrimaryKeyRequired` and `ErrModelValueRequired` are.

Reply:
@ExerciseBook This has nothing to do with the primary key, it's because there aren't any accessible fields in the model.
The use of `ErrModelValueRequired` here is indeed misleading. It prompts the user to use the model, but the problem here is that the user uses the wrong model.